sdk/tests/deqp/functional/gles3/es3fFboTestCase.js

```diff
@@ -135,7 +135,7 @@ var DE_ASSERT = function(x) {
             /** @const @type {goog.NumberArray} */ var supportedSampleCounts = es3fFboTestCase.querySampleCounts(sizedFormat);
             var supported = Array.prototype.slice.call(supportedSampleCounts);
             if (supported.indexOf(numSamples) == -1) {
-                if (numSamples >= 2) {
+                if (minSampleCount == 0 || numSamples > gl.getParameter(gl.MAX_SAMPLES)) {
```

Comment:
Per my understanding, R11F_G11F_B10F, RG16F, and R16F are totally optional regarding the supported samples. So for these 4 formats, it's okay to pass the test if numSamples isn't supported. But RGBA32F, RGBA16F and RG32F are partially optional. They must support those samples which are no more than MAX_SAMPLES. So for these 3 formats, we must fail the test if this's not obeyed. Furthermore if numSamples is more than MAX_SMAPLES, the support is optional, so we can pass the test with "Sample count not supported".